typescript/audio-summarizer/README.md

```diff
@@ -0,0 +1,188 @@
+# Emojifier 🤖🎨 - A simple audio summarizer with observability on Literal AI
+
+In this cookbook, we will start with a simple audio summarizer using the Web Speech API and Next.js. We will then add observability to the application using Literal AI.
```

Comment:
Ideally the intro would metion the role of all tools used and how they are related, followed by a bullet point that extends of each:
- Next.js
- OpenAI Whisper
- OpenAi GPT-4
- Literal AI

Reply:
done, i reworked the intro paragraph tell me what you think !